src/env-inl.h

```diff
@@ -617,8 +617,8 @@ void Environment::CreateImmediate(native_immediate_callback cb,
   native_immediate_callbacks_.push_back({
     cb,
     data,
-    std::unique_ptr<Persistent<v8::Object>>(obj.IsEmpty() ?
-        nullptr : new Persistent<v8::Object>(isolate_, obj)),
+    obj.IsEmpty() ? v8::Global<v8::Object>() :
+        v8::Global<v8::Object>(isolate_, obj),
```

Comment:
The ternary isn't necessary; calling the constructor with an empty handle leaves it empty.

Reply:
Thanks, done!